presentation/common/src/presentation-common/content/PropertyValueFormatter.ts

```diff
@@ -120,21 +120,26 @@ export class ContentPropertyValueFormatter {
         }
       : async (rawValue: number) => formatDouble(rawValue);
 
-    return this.formatValue(field.type, value, { doubleFormatter });
+    return this.formatValue(field.type, value, field, { doubleFormatter });
```

Comment:
doesn't look like we need to pass `field.type` anymore, since we're now passing `field`

Reply:
Removed type where type is generic.